src/Razor/test/VSCode.FunctionalTest/tests/Rename.test.ts

```diff
@@ -47,6 +53,7 @@ suite('Rename', () => {
     test('Can rename symbol within .cshtml', async () => {
         const cshtmlDoc = await vscode.workspace.openTextDocument(cshtmlPath);
         const cshtmlEditor = await vscode.window.showTextDocument(cshtmlDoc);
+        await new Promise(r => setTimeout(r, 5000));
```

Comment:
My theory is that the first test in a test file is more likely to fail because that project/whatever is being initialized. So I'm hardening this test as well as part of the PR.